packages/svelte/elements.d.ts

```diff
@@ -892,7 +892,7 @@ export interface HTMLButtonAttributes extends HTMLAttributes<HTMLButtonElement>
 		| 'text/plain'
 		| undefined
 		| null;
-	formmethod?: 'dialog' | 'get' | 'post' | undefined | null;
+	formmethod?: 'dialog' | 'get' | 'post' | 'DIALOG' | 'GET' | 'POST' | undefined | null;
```

Comment:
should we also include capitalized variations? I believe they are also valid?
```suggestion
	formmethod?: 'dialog' | 'get' | 'post' | 'DIALOG' | 'GET' | 'POST' | 'Dialog' | 'Get' | 'Post' | undefined | null;
```

Reply:
Are they that common though? Searching through github and it seems to only be lowercase or all uppercase that I can find.

Reply:
I think it is common practice to see lower case only or upper case only, and I wouldn't mind even having only one version for consistency. Otherwise auto-completion will look like a mess if we respect the case-insensitivity 
<img width="872" alt="Capture d’écran 2024-10-17 à 11 21 14" src="https://github.com/user-attachments/assets/6c81b572-88b0-4d3b-9eff-9bee3636ea2b">